recipes/flatcc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.12)
+project(flatcc_example)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(INC_DIR "${PROJECT_SOURCE_DIR}/include")
+set(GEN_DIR "${CMAKE_CURRENT_BINARY_DIR}/generated")
+set(FBS_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
+
+include_directories("${GEN_DIR}" "${INC_DIR}")
+
+add_custom_target(gen_monster_fbs ALL)
+add_custom_command (
+    TARGET gen_monster_fbs
+    COMMAND cmake -E make_directory "${GEN_DIR}"
+    COMMAND flatcc -a -o "${GEN_DIR}" "${FBS_DIR}/monster.fbs"
+    DEPENDS flatcc "${FBS_DIR}/monster.fbs"
```

Comment:
Looks like (Macos) debug creates flatcc_d executable.

Reply:
Yes, I tested only with Release settings. Will fix that tomorrow.